Gemfile.lock

```diff
@@ -316,7 +316,7 @@ DEPENDENCIES
   selenium-webdriver
   sidekiq (~> 6.0)
   standard
-  stimulus_reflex (= 3.2.3)
+  stimulus_reflex (= 3.3.0pre1)
```

Comment:
This is just to make it work without calling params explicitely: see https://github.com/hopsoft/stimulus_reflex/pull/249